lib/scopes.js

```diff
@@ -0,0 +1,29 @@
+var SCOPES = {
+  user: [
```

Comment:
So, What we currently have available from login.wmo is:

``` json
{
    "avatar": "https://secure.gravatar.com/avatar/5410ec660422c8ea12d4a28ef794a23a?d=https%3A%2F%2Fstuff.webmaker.org%2Favatars%2Fwebmaker-avatar-200x200.png",
    "email": "chris@chrisdecairos.ca",
    "emailHash": "5410ec660422c8ea12d4a28ef794a23a",
    "id": 1,
    "isAdmin": false,
    "isMentor": false,
    "isSuperMentor": false,
    "prefLocale": "en-US",
    "sendEventCreationEmails": true,
    "sendCoorganizerNotificationEmails": true,
    "sendMentorRequestEmails": true,
    "username": "cade"
  }
```
how about id, isAdmin, isMentor, isSuperMentor, & prefLocale,